dartagnan/src/main/java/com/dat3m/dartagnan/configuration/OptionNames.java

```diff
@@ -64,6 +65,8 @@ public class OptionNames {
 
     // Refinement Options
     public static final String BASELINE = "refinement.baseline";
+    public static final String GRAPHVIZ_DEBUG_FILES = "refinement.generageGraphvizDebugFiles";
+    public static final String SYMMETRY_LEARNING = "refinement.symmetryLearning";
```

Comment:
The option was named `symmetricLearning` and not `symmetryLearning` (it used to have the latter name).
I changed it a while ago because the latter sounds like we learn the symmetries, but that is not true: we use the symmetries to learn symmetric clauses.